rtt_rosclock/src/rtt_rosclock.cpp

```diff
@@ -55,13 +55,17 @@ const ros::Time rtt_rosclock::rtt_now()
   const uint64_t one_E9 = 1000000000ULL;
   // NOTE: getNSecs returns wall time, getTicks returns offset time
 #ifdef __XENO__
```

Comment:
Why is this distinction between Xenomai and non-Xenomai builds required here at all, independent of sim time?
Wouldn't `getTicks()` (the else branch) work for both cases?

`getTicks()` => `rt_timer_tsc()` (if system clock is enabled in the TimeService)
`getNSecs()` => `rt_timer_ticks2ns(rt_timer_read())` (independent of sim/system time mode)